drive/tests/query_tests.rs

```diff
@@ -170,7 +159,7 @@ fn test_query() {
     let query_value = json!({
         "where": [
         ],
-        "limit": 100,
+        "limit": 100i64,
```

Comment:
it's unsigned

Reply:
this is weird, as compiler shouted at me because it couldn't decide which implementation of `Serialize` to use so I had to add these type hints, but now I removed them and it works like nothing happened